build-tools/installers/create-installers.targets

```diff
@@ -0,0 +1,122 @@
+    <MSBuildFiles Include="$(MSBuildSrcDir)\class-parse.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\desugar_deploy.jar"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\FSharp.Compiler.CodeDom.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\FSharp.Core.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\generator.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\illinkanalyzer.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Irony.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\javadoc-to-mdoc.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.dll.config"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.Export.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.Tools.Cecil.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.Tools.Diagnostics.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Interop.Tools.JavaCallableWrappers.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Runtime.Environment.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Java.Runtime.Environment.dll.config"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\jcw-gen.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\jnimarshalmethod-gen.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\LayoutBinding.cs"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\libZipSharp.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\libZipSharp.dll.config"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\logcat-parse.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\mdoc.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\mkbundle.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\mkbundle-api.h"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\mono-symbolicate.exe"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.CSharp.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Options.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Terminal.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\monodoc.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\MULTIDEX_JAR_LICENSE"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\**\proguard.jar"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\**\license.html"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\r8.jar"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\SgmlReaderDll.dll"></MSBuildFiles>
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin*"></MSBuildFiles>
+  </ItemGroup>
+  <ItemGroup> <!-- monodroid -->
```

Comment:
For "cleanliness" (?), we could pick *one* file from the `Condition` checks below, and make the `<ItemGroup/>` *itself* conditional, and remove the other conditions:

```xml
<ItemGroup Condition="Exists('$(MSBuildSrcDir)\Mono.AndroidTools.dll')">
  <MSBuildFiles Include="$(MSBuildSrcDir)\ICSharpCode.SharpZipLib.dll"></MSBuildFiles>
  <MSBuildFiles Include="$(MSBuildSrcDir)\INIFileParser.dll"></MSBuildFiles>
  <MSBuildFiles Include="$(MSBuildSrcDir)\jar2xml.jar"></MSBuildFiles>
  <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.AndroidTools.dll"></MSBuildFiles>
  <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Android.DebugRuntime-*"/>	
</ItemGroup>
```

This would also help insure that when we're doing a commercial build, we *require* all of these files, as opposed to just "requiring" whatever happens to exist.

Reply:
This has been addressed, though I am going to look to clean up wildcards in this file shortly.